setup.sh

```diff
@@ -36,7 +36,9 @@ while [[ $# -gt 0 ]]; do
 done
 
 if [[ -z "$repos" ]]; then
-  repos="worker server jobs pgbouncer data-model"
+  repos="worker server jobs pgbouncer data-model stack-analysis \
```

Comment:
Not sure this is worth it. We are curling the templates directly from GitHub, we don't need all the repos cloned locally.

Reply:
I think that are leftovers from docker-compose local deployment.

Reply:
I'd say not add these. We don't reference these services from the main docker-compose file, so cloning them is not necessary. Whoever wants to work on those services can clone them separately.